config/locales/de.yml

```diff
@@ -279,25 +279,25 @@ de:
     title: "Student Guide"
     attendance_policy:
       title: "Attendance Policy"
-      intro: "codebar is, and always has been, open to all levels of coding experience. While most of the tutorials on our website are aimed at newcomers to web development, we intend to continue our open-door policy for any student who needs coach support on their own projects. Due to the fact that our workshops are frequently oversubscribed, we ask students to respect the following expectations for participation in our workshops"
-      rule1: "In order to attend a workshop, <u>you must be on the RSVP list.</u> If you have not RSVP’d or were not allocated a spot, you will not be admitted into a workshop."
-      rule2: "If you can no longer attend a workshop, <u>please cancel your RSVP by 12:00 latest </u> on the day of the workshop. We understand that last-minute emergencies occur -- if this is the case, please get in touch with an organizer directly as soon as possible. Repeated no-shows will result in temporary or permanent suspension of your invitations."
-      rule3: "No guests are allowed. Our workshops are already overcrowded and we cannot make any exceptions. If your friend meets our eligibility criteria, get them to sign up and RSVP themselves."
-      rule4: "Please be respectful of our hosts. Do not show up exceedingly early to a workshop (not earlier than 15 minutes before the event begins) and try to leave the workstation clean and as you found it."
-      rule5: "<u>Please do not work on projects that were assigned as part of any bootcamp or university curriculum.</u> Talk to your course adviser if you feel like you need additional support with your coursework. We welcome your attendance, but we encourage you to use your time at codebar to develop skills that are outside the normal scope of your studies."
-      rule6: "Do not bring along and ask for help with any commercial work. This involves work you're being paid to do."
-      rule7: "Projects that involve specific frameworks/libraries (Rails, NodeJS, Meteor, Angular, Ember, React, etc) or languages that are not part of the codebar tutorials (anything other than Ruby or Javascript) are welcome but we cannot guarantee that a coach will be available to help you. In these cases we recommend selecting a codebar tutorial or katas (codewars.com or exercism.io) as a back-up option."
+      intro: "codebar ist und war schon immer offen für alle Arten von Programmiererfahrung. Die meisten Tutorials auf unserer Website richten sich zwar an Einsteiger in die Webentwicklung, aber wir wollen unsere Politik der offenen Tür für alle Studenten fortsetzen, die bei ihren eigenen Projekten Unterstützung durch einen Coach benötigen. Da unsere Workshops häufig überzeichnet sind, bitten wir die Studenten, die folgenden Erwartungen an die Teilnahme an unseren Workshops zu erfüllen"
```

Comment:
Not sure if `levels` translates well to `Arten`. Maybe `Stufen`?

`Politik` might be a bit odd. Maybe `Türen für Alle offen halten`